common/src/main/java/host/bloom/ab/common/config/ConfigKeys.java

```diff
@@ -0,0 +1,47 @@
+package host.bloom.ab.common.config;
+
+import ch.jalu.configme.Comment;
+import ch.jalu.configme.SettingsHolder;
+import ch.jalu.configme.configurationdata.CommentsConfiguration;
+import ch.jalu.configme.properties.Property;
+import host.bloom.ab.common.config.enums.BlockNewJoins;
+import host.bloom.ab.common.config.enums.Location;
+import org.jetbrains.annotations.NotNull;
+import static ch.jalu.configme.properties.PropertyInitializer.newBeanProperty;
+import static ch.jalu.configme.properties.PropertyInitializer.newProperty;
+
+public class ConfigKeys implements SettingsHolder {
+
+    protected ConfigKeys() {}
+
+    @Override
+    public void registerComments(@NotNull CommentsConfiguration conf) {
+        String[] header = {
+                "If you have any questions, Please contact us at:",
+                "",
+                "Support: discord.gg/bloom"
```

Comment:
Just in case someone doesn't have Discord
```suggestion
                "If you have any questions, Please contact us at:",
                "https://bloom.host/support"
```